src/interfaces/index.ts

```diff
@@ -32,6 +32,10 @@ export interface PassengerDetails {
     proof?: string[];
 }
 
+export interface SelectSalesOfferPackageWithError {
+    errorMessage: string;
+}
```

Comment:
Should this `SelectSalesOfferPackageWithError` interface live on the page/api where it is used? I think most of these kinds of interfaces (which are only used for one page/api) are stored this way

Reply:
Also, should this not be 
```
export interface SelectSalesOfferPackageWithError {
    errors: ErrorInfo[];
}
```

Reply:
I have moved the interface to SelectSalesOfferPackage.ts. I haven't change it to comprise a ErrorInfo[] under a key of errors as we only have a single error message on SelectSalesOfferPackage.ts.